gr-digital/python/digital/bindings/crc16_async_bb_python.cc

```diff
@@ -15,7 +19,11 @@
 /* BINDTOOL_GEN_AUTOMATIC(0)                                                       */
 /* BINDTOOL_USE_PYGCCXML(0)                                                        */
 /* BINDTOOL_HEADER_FILE(crc16_async_bb.h)                                        */
+<<<<<<< HEAD
```

Comment:
Merge markers

Reply:
Thanks you @daniestevez. Im not used to use git and things like this are a headhache for me.